extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -817,6 +811,15 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     } else {
       size = this.getSlotSize();
     }
+    // If this is a multi-size creative, fire delayed impression now. If it's
+    // fluid, wait until after resize.
+    if (this.isFluid_ && !this.returnedSize_) {
```

Comment:
I realize this is somewhat outside the scope of this PR, but if this.isFluid_ can be true even when it is not fluid as you say in your description, I think you should change it to an accurate name.

Reply:
Changed to `isFluidRequest_`, which is more accurate.